### ISSUE TITLE
Test CephFS usage 

### ISSUE THREAD

**Issue by mkyc** [epiphany-platform/epiphany#1571]
TODO

look [here](https://github.com/epiphany-platform/epiphany/issues/1449#issuecomment-676505931)

**Comment by rpudlowski93** [comment]
As the description says, **Block Storage** allows a single pod to mount storage in RWO mode and when there will be a crash for example node will break or will be unavailable (shutdown) the pod will go into terminating state and application will be unavailable : 

Error: 
`Unable to attach or mount volumes: unmounted volumes=[wordpress-persistent-storage], unattached volumes=[wordpress-persistent-storage default-token-4nj25]: timed out waiting for the condition`

Moreover, it is not possible to run deployment and have clustered application with PVC and Block Storage, due to the same error as above. 

In this case, [**CephFS** ](https://rook.io/docs/rook/v1.4/ceph-filesystem.html)is much more better option and after crash, Pod will recreate after 5 minutes (default eviction time can be changed) and application will have only short downtime. Moreover, in compare to Block Storage, deployments and replicas work as expected since shared filesystem can be mounted with read/write permission from multiple pods . 

Summing up, for Epiphany usage, I could highly recommend to use Block Storage in StatefulSets since StatefulSets allow to use volumeClaimTemplates and problem with PVC in RWO mode doesn't affect our clustered application, it could be nice solution for database. Block volumes are critical to applications like databases (MongoDB, Cassandra, PostgreSQL) that require consistent I/O performance and low latency. As far as CephFS is consider, I would highly recommend to use this kind of storage for clustered application using deployment with multiple pods.

**Comment by rpudlowski93** [comment]
Nothing to do here more, tomorrow demo will be presented
